_templates/exchanges.html

```diff
@@ -24,6 +24,8 @@ <h2 class="exchanges-tab-title exchanges-international-title" id="international"
 <div class="toccontent-block boxexpand expanded">
   <h2 class="exchanges-tab-title exchanges-p2p-title" id="p2p">Peer-to-Peer (P2P)</h2>
   <p>
+    <a class="marketplace-link" href="https://bitquick.co/">BitQuick</a>
```

Comment:
Exchanges are listed alphabetically; should be listed after Bisq.

Reply:
Sorry about that - updated PR

https://github.com/bitcoin-dot-org/bitcoin.org/pull/2674/commits/237294a1c24d47246e8e1cc109bb2cb28f6ba995